crates/turborepo-lib/src/config.rs

```diff
@@ -508,6 +534,34 @@ impl TurborepoConfigBuilder {
         Ok(local_config)
     }
 
+    fn get_global_auth(&self) -> Result<ConfigurationOptions, Error> {
```

Comment:
Don't we want this logic in the auth crate?

Reply:
We do, but this is one of those stopgap fixes. I plan on moving most, if not all, of this kinda logic into the auth crate in a subsequent PR